README.md

```diff
@@ -572,14 +567,168 @@ Here's how you can configure it:
 
 This configuration option may not be changed after installation.
 
+### Recovery and Health Checks
+
+You can enable automated replacement of brokers and configure the circumstances under which they are replaced.
+
+#### Enable Broker Replacement
+
+To enable automated replacement: <!-- what's all this about data loss, then? -->
+
+* **DC/OS CLI options.json**:
+
+```json
+    {
+        "enable_replacement":{
+            "description":"Enable automated replacement of Brokers. WARNING: May cause data loss. See documentation.",
```

Comment:
Kafka doesn't really have backup.  It has mirror maker.  The warning is this.

"The replacement mechanism has no way of knowing if the Broker it is destructively replacing had the last copy of some data.  So depending on a user's specified replication policy and the degree and duration of the permanent failures the system has encountered, data loss is a possibility."